MarkupsToModel/MRML/vtkMRMLMarkupsToModelNode.cxx

```diff
@@ -48,19 +36,19 @@ vtkMRMLMarkupsToModelNode::vtkMRMLMarkupsToModelNode()
   events->InsertNextValue( vtkCommand::ModifiedEvent );
   events->InsertNextValue( vtkMRMLMarkupsNode::PointModifiedEvent );//PointEndInteractionEvent
 
-  this->AddNodeReferenceRole( MARKUPS_ROLE, NULL, events.GetPointer() );
-  this->ModelNodeID = "";
-  this->ModelNode = NULL;
-
-  this->AutoUpdateOutput=true;
-  this->CleanMarkups=true;
-  this->ConvexHull=true;
-  this->ButterflySubdivision=true;
-  this->DelaunayAlpha=0.0;
-  this->TubeRadius=1.0;
+  this->AddNodeReferenceRole( INPUT_MARKUPS_ROLE, NULL, events.GetPointer() );
+  this->AddNodeReferenceRole( OUTPUT_MODEL_ROLE );
+
+  this->AutoUpdateOutput = true;
+  this->CleanMarkups = true;
+  this->ConvexHull = true;
+  this->ButterflySubdivision = true;
+  this->DelaunayAlpha = 0.0;
+  this->TubeRadius = 1.0;
+  this->TubeResolutionLength = 20;
+  this->TubeResolutionAround = 20;
```

Comment:
TubeResolutionAround is not a very clear name. Around what? Can you find a better name?